mesheryctl/internal/cli/root/system/channel.go

```diff
@@ -81,11 +82,62 @@ var viewCmd = &cobra.Command{
 	},
 }
 
+//check release channel version string supplied by user for validity. This func does not set any thing.(Just for checks)
+func validVersionCheck(c string) (bool, error) {
```

Comment:
```suggestion
func validateChannel(c string) (bool, error) {
```

Reply:
Also it would be good to shift this as central function so it could be used again.